requirements.in

```diff
@@ -7,3 +7,4 @@ pytest
 pytest-cov
 Pygments
 six
+whatthepatch-graingert
```

Comment:
Would prefer to use vanilla whatthepatch, yuo haven't even given @cscorley 24 hours to merge your PR's like cscorley/whatthepatch#13